spec/models/manageiq/providers/openshift/container_manager/refresher_spec.rb

```diff
@@ -185,18 +197,18 @@ def normal_refresh
       it "archives objects" do
         expect(ContainerProject.count).to eq(8)
         expect(ContainerProject.where(:deleted_on => nil).count).to eq(7)
-        expect(ContainerImage.count).to eq(43) # should be 44
-        expect(ContainerImage.where(:deleted_on => nil).count).to eq(43) # should be 44
+        expect(ContainerImage.count).to eq(38) # should be 39
+        expect(ContainerImage.where(:deleted_on => nil).count).to eq(38) # should be 39
```

Comment:
@cben this is the image form the pods `my-pod-$X` that are all deleted. because it is not saved in the openshift registry I guess we are not persisting it in any way after all references to it are gone.

The image name is "jboss-decisionserver-6/decisionserver63-openshift"